src/ddmd/traits.d

```diff
@@ -403,6 +404,50 @@ extern (C++) Expression semanticTraits(TraitsExp e, Scope* sc)
         return new IntegerExp(e.loc, false, Type.tbool);
     }
 
+    /**
+     Params: o: the AST-node to check.
+     Params: fdp: optionally a pointer to a function declararion,
+        which will get set o is a function declarartion.
+
+     Returns: a function-type if
+        o is a declarion of a delegate, a function,
+            a function pointer
+            or a variable of those types,
+        it will return the functionType
+     else
+        it will return null.
+    */
```

Comment:
I offer you this documentation header:

```D
/**
 Gets the `TypeFunction` for `o`
 Params:
    o   = the AST node to get the `TypeFunction` for
    fdp = optional pointer to a function declararion, to be set 
          if `o` is a function declarartion
 Returns: 
    a `TypeFunction` if `o` is a declaration for a delegate, function,
    or avariable of those types.  Otherwise, `null`
*/
```

Reply:
Neat Thanks!